eternalcore-api/src/main/java/com/eternalcode/core/feature/jail/JailedPlayer.java

```diff
@@ -0,0 +1,48 @@
+package com.eternalcode.core.feature.jail;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+
+public class JailedPlayer {
```

Comment:
Prisoner

Reply:
I hope this message finds You well,

The whole idea of name migration from `Prisoner` to `JailedPlayer` came to me and Mr. Norbert from package and feature name. We went through variety of different names and concluded with the one matching feature name. As You can see the name reflects Players in prison, that's why your idea comes to mind thirst. Although it's correct, I think matching both:
- lite commands name,
- feature name,
it's more important than simplifying mentioned class name.

If You agree with me resolve this conversation. If You have any questions feel free to comment them below.

Yours sincerely,
Citral

Reply:
Then why does the rest of the code rely on the name "Prisoner"? For example PrisonerRepository?

Reply:
So what does this interface do? It contains methods like savePrisoner, so what does this method do? Does it save a Prisoner or a JailedPlayer? I still think that Prisoner is a better name, it's shorter and more understandable at first glance. I don't know why Mr. Norbert would disagree with that.

![image](https://github.com/EternalCodeTeam/EternalCore/assets/65517973/b453cb0d-1c68-4fc7-a960-5c948c37323c)

Reply:
To jest implementacja a nie API. Wejdź w IntelliJ i zobacz sobie jak to wygląda po pakietach etc

Reply:
I don't understand this solution, but I believe it's better.